Loaded module: python3/3.6.2
Traceback (most recent call last):
  File "./tools/train.py", line 255, in <module>
Traceback (most recent call last):
  File "./tools/train.py", line 255, in <module>
    default_setup(args)
  File "/zhome/f9/4/160189/SegmenTron/segmentron/utils/default_setup.py", line 24, in default_setup
    default_setup(args)
  File "/zhome/f9/4/160189/SegmenTron/segmentron/utils/default_setup.py", line 24, in default_setup
    torch.cuda.set_device(args.local_rank)
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/cuda/__init__.py", line 311, in set_device
    torch.cuda.set_device(args.local_rank)
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/cuda/__init__.py", line 311, in set_device
    torch._C._cuda_setDevice(device)
RuntimeError: CUDA error: invalid device ordinal
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.    torch._C._cuda_setDevice(device)

RuntimeError: CUDA error: invalid device ordinal
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/launch.py:186: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use_env is set by default in torchrun.
If your script expects `--local_rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  FutureWarning,
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 21679 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 21680 closing signal SIGTERM
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 2 (pid: 21681) of binary: /zhome/f9/4/160189/IGR/data/torch/bin/python
Traceback (most recent call last):
  File "/appl/python/3.6.2/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/appl/python/3.6.2/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/launch.py", line 193, in <module>
    main()
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/launch.py", line 189, in main
    launch(args)
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/launch.py", line 174, in launch
    run(args)
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/run.py", line 713, in run
    )(*cmd_args)
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/launcher/api.py", line 131, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/zhome/f9/4/160189/IGR/data/torch/lib/python3.6/site-packages/torch/distributed/launcher/api.py", line 261, in launch_agent
    failures=result.failures,
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
./tools/train.py FAILED
------------------------------------------------------------
Failures:
[1]:
  time      : 2022-02-28_17:06:48
  host      : n-62-11-15.hpccluster.dtu.dk
  rank      : 3 (local_rank: 3)
  exitcode  : 1 (pid: 21682)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2022-02-28_17:06:48
  host      : n-62-11-15.hpccluster.dtu.dk
  rank      : 2 (local_rank: 2)
  exitcode  : 1 (pid: 21681)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
